custom_components/rivian/const.py

```diff
@@ -480,6 +480,17 @@
             state_class=SensorStateClass.MEASUREMENT,
             suggested_display_precision=0,
         ),
+        RivianSensorEntityDescription(
+            key="altitude",
+            field="gnssAltitude",
+            name="Altitude",
+            icon="mdi:altimeter",
+            device_class=SensorDeviceClass.DISTANCE, 
+            native_unit_of_measurement=UnitOfLength.METERS,
+            state_class=SensorStateClass.MEASUREMENT,
+            suggested_display_precision=1,
+            suggested_unit_of_measurement=UnitOfLength.MILES,
```

Comment:
I think FEET would be more appropriate here for the altitude. That being said, we can remove the `suggested_unit_of_measurement` altogether as HA will automatically default to the user's locale preference.

Reply:
If we stick with meter/feet, we could also probably set the display precision to 0